flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/IntermediateResultInfo.java

```diff
@@ -35,6 +35,14 @@ public interface IntermediateResultInfo {
      */
     boolean isBroadcast();
 
+    /**
+     * Indicates whether every downstream consumer needs to consume all produced sub-partitions.
```

Comment:
I see this method seems to be relate to broadcast - it would be good to explain how.